<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mask_management</class>
 <widget class="QWidget" name="mask_management">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="gb_text">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>文本信息录入</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QWidget" name="wid_texttal" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextEdit" name="textEdit_2"/>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QComboBox" name="comboBox_4">
            <item>
             <property name="text">
              <string>部件1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>部件2</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_3"/>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_5">
            <item>
             <property name="text">
              <string>工序1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>工序2</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_ok">
            <property name="text">
             <string>确定</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="gb_other">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>其他信息录入</string>
   </property>
   <widget class="QWidget" name="wid_choosefile" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>231</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QWidget" name="widget_2" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="btn_choosefile">
          <property name="text">
           <string>选取文件</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QTextEdit" name="textEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="comboBox_2">
          <item>
           <property name="text">
            <string>部件1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>部件2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>部件3</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_2"/>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_3">
          <item>
           <property name="text">
            <string>工序1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>工序2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>工序3</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>图片信息</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_menuok">
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>256</width>
     <height>131</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>维修对象</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>部件1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>部件2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>工序1-1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>工序1-2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>1-1文本</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>1-1工艺信息</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>1-1图片信息</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>1-1零件信息</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>减速器</string>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>发射单元</string>
    </property>
   </item>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>20</y>
     <width>111</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>1</string>
    </property>
   </column>
   <item>
    <property name="text">
     <string>减速器</string>
    </property>
    <item>
     <property name="text">
      <string>部件1</string>
     </property>
     <item>
      <property name="text">
       <string>工序1</string>
      </property>
      <item>
       <property name="text">
        <string>文本</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>零件</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>工艺</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>图片</string>
       </property>
      </item>
     </item>
     <item>
      <property name="text">
       <string>工序2</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>部件2</string>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>发射单元</string>
    </property>
    <item>
     <property name="text">
      <string>部件1</string>
     </property>
    </item>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
